packages/ipfs-core/src/components/pin/pin-manager.js

```diff
@@ -57,7 +66,7 @@ class PinManager {
         yield * this._walkDag(link.Hash, { preload })
       }
     } else if (cid.codec === 'dag-cbor') {
-      for (const [_, childCid] of dagCborLinks(node)) { // eslint-disable-line no-unused-vars
+      for (const [, childCid] of dagCborLinks(node)) { // eslint-disable-line no-unused-vars
```

Comment:
Is the eslint comment still necessary with this change?

Reply:
```suggestion
      for (const [, childCid] of dagCborLinks(node)) {
```